examples/seasonal-journey/service.bal

```diff
@@ -0,0 +1,80 @@
+// AUTO-GENERATED FILE. DO NOT MODIFY.
+// This file is auto-generated by the Ballerina OpenAPI tool.
+
+// Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/log;
+import ballerinax/salesforce.marketingcloud as mc;
+
+configurable string clientId = ?;
+configurable string clientSecret = ?;
+configurable string subDomain = ?;
+configurable string accountId = ?;
+
+type User record {
+    string userId;
+    string email;
+    string firstName;
+    string signupSource;
+};
+
+mc:Client marketingCloudClient = check new (
```

Comment:
Should not this client be initialized inside the HTTP service. AFAIR that is the practice we follow with listener-service based applications.

Reply:
Fixed in [b66820b](https://github.com/ballerina-platform/module-ballerinax-salesforce.marketingcloud/pull/2/commits/b66820bdfebf8d744d52182ba69342eef29e66be)